invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/doi.html

```diff
@@ -6,7 +6,8 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{%- set id_doi = metadata.identifiers|doi_identifier %}
+{%- set id_doi = record.pids|doi_identifier %}
+<dt><b>{{_('DOI')}}</b></dt>
 <dd>
```

Comment:
There is a tiny space that does not show here. See e.g. the subjects title gets a bit of space in between title and content. It is because it lacks the css class (it was already "broken", not coming from your changes but we can fix it now since you are touching the file).

```suggestion
<dd class="top-bottom-padded">
```